po/ru.po

```diff
@@ -122,51 +133,53 @@ msgstr "Выражение `if`"
 
 #: src/SUMMARY.md src/control-flow-basics/loops.md
 msgid "Loops"
-msgstr ""
+msgstr "Циклы"
 
 #: src/SUMMARY.md src/control-flow-basics/loops/for.md
 msgid "`for`"
-msgstr ""
+msgstr "`for`"
```

Comment:
@mgeisler Does this actually need to be "translated"? The `msgid` and `msgstr` are exactly the same.

Reply:
Making them the same is the correct way to say `for` should be remain unchanged as `for`.

You're correct that leaving the entry empty will have the same effect — but the entry will then show up as untranslated in various statistics. So it's better to do what @1dimir did here and translate it to the same thing.